lib/main.dart

```diff
@@ -1,87 +1,20 @@
-import 'dart:io';
-
 import 'package:flutter/material.dart';
+import 'package:worshipsongs_app/views/TabBarWidgets.dart';
 
 import 'db/DatabaseHandler.dart';
+import 'domain/Author.dart';
+import 'domain/Song.dart';
 
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
-  List<Map> songs = await DatabaseHandler().retrieveAttributes('SELECT title, comments FROM songs', 'title');
-  runApp(MyApp(
-    songs: List<Map>.generate(songs.length, (index) => songs[index]),
+  List<Song> songs = await DatabaseHandler().findAllSongs();
+  List<Author> authors = await DatabaseHandler().findAllAuthors();
+  List<Map> books = await DatabaseHandler().retrieveAttributes('select sb.name, count(ssb.song_id) as songs from song_books sb, songs_songbooks ssb where sb.id = ssb.songbook_id group by sb.name', 'name');
+  List<Map> topics = await DatabaseHandler().retrieveAttributes('select t.name, count(st.song_id) as songs from topics t, songs_topics st where t.id = st.topic_id group by t.name', 'name');
```

Comment:
can we create a domain in name of topics with only name and id. and later in that view we will execute and get the list of songs from it by song_topics ID.

Reply:
Removed this for now. When we add the topics then we will add it.